sdk/python/lib/pulumi/resource.py

```diff
@@ -206,9 +204,10 @@ def collapse_alias_to_urn_worker(inner: Union[Alias, str]) -> 'Output[str]':
 
         return create_urn(name, type_, parent, project, stack)
 
-    inputAlias: Output[Union[Alias, str]] = Output.from_input(alias)
+    inputAlias: 'Output[Union[Alias, str]]' = Output.from_input(alias)
```

Comment:
Why does this need to be quoted? Haven't we imported `Output` by this point?

Reply:
(same throughout)

Reply:
ah yup totally